src/cli/commands/init/init.ts

```diff
@@ -130,6 +136,17 @@ export async function init(options: SWACLIConfig, showHints: boolean = true) {
 
     logger.log(`- Use ${chalk.cyan("swa deploy")} to deploy your app to Azure.\n`);
   }
+  const end = new Date().getTime();
+
+  await collectTelemetryEvent(TELEMETRY_INIT_EVENT, {
+    macAddressHash: (await getMachineId("sha256", TELEMETRY_MAC_ADDRESS_HASH_LENGTH)).toString(),
+    subscriptionId: DEFAULT_CONFIG.subscriptionId!,
+    sessionId: getSessionId(end),
```

Comment:
looks little or for us to take end time as the sessionId.. Any reason for that.. Also can it be calculated in the telemetry util or during the initialization of the telemetry library. Session is a uniqueid for the cli invocation?

Reply:
Yes, SessionId is the unique Id for each time CLI is invoked. we are calculating it as a combination of ProcessId and timeStamp and get the hash of the combined string. Used `end` as the timestamp as I wanted to calculate the session Id at the time when the call to collecttelemetryEvent is made.